src/framework/components/camera/component.js

```diff
@@ -556,6 +568,51 @@ Object.assign(pc, function () {
             } else {
                 callback("Not presenting VR");
             }
+        },
+
+        /**
+         * @function
+         * @name pc.CameraComponent#startXr
+         * @description Attempt to start XR session with this camera
+         * @param {string} type - The type of session. Can be one of the following:
+         *
+         * * {@link pc.XR_TYPE_INLINE}: Inline - always available type of session. It has limited features availability and is rendered into HTML element.
+         * * {@link pc.XR_TYPE_IMMERSIVE_VR}: Immersive VR - session that provides exclusive access to VR device with best available tracking features.

```

Comment:
In the interests of a simpler API, what about: `pc.XR_TYPE_IMMERSIVE_VR` -> `pc.XRTYPE_VR`? Same for AR. (BTW, the general naming convention is `pc.THISISTHETYPE_THISISTHESUBTYPE`, so one underscore. Yes, I know some enums break this).